transmogrifier/sources/xml/ead.py

```diff
@@ -557,6 +493,96 @@ def get_publishers(cls, source_record: Tag) -> list[timdex.Publisher] | None:
             return [timdex.Publisher(name=publication_value)]
         return None
 
+    @classmethod
+    def get_related_items(cls, source_record: Tag) -> list[timdex.RelatedItem] | None:
+        related_items = []
+        collection_description = cls._get_collection_description(source_record)
+
+        for related_item_element in collection_description.find_all(
```

Comment:
Can't recall what the convention was in this file, but could optionally just iterate over the method call directly.

Reply:
FWIW, I've generally been iterating over a variable when a `.find_all(...)` or other call is needed for iteration , `for related_item_element in collection_description.find_all()` rather than `for related_item_element in cls._get_collection_description(source_record).find_all()`

Reply:
Ah, yes! I've been doing the same as @ehanson8. If it is followed by a `.find`/`.findall`, assign to a variable! Feels a little easier to read.